source/features/warn-pr-from-master.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'dom-chef';
-import {$, elementExists} from 'select-dom';
+import {$} from 'select-dom';
 import * as pageDetect from 'github-url-detection';
 
 import features from '../feature-manager.js';
 import getDefaultBranch, {defaultBranchOfRepo} from '../github-helpers/get-default-branch.js';
 import {getRepo} from '../github-helpers/index.js';
+import observe from '../helpers/selector-observer.js';
 
-async function init(): Promise<false | void> {
+async function warnPR(crossRepo: Element): Promise<false | void> {
```

Comment:
This this is no longer an `init` function, it doesn't need to return `false` anymore`, it can just `return;`

```suggestion
async function warnPR(crossRepo: Element): Promise<void> {
```